op-node/rollup/driver/sequencer.go

```diff
@@ -89,198 +90,198 @@ func NewSequencer(log log.Logger, rollupCfg *rollup.Config, engine derive.Engine
+// 	// derivation pipeline.
+
+// 	l1Origin, err := derive.NodeKitL1Origin(ctx, d.rollupCfg, &sysCfg, batch.onto,
+// 		batch.jst.Next.L1Head, d.l1OriginSelector, d.log)
+// 	if err != nil {
+// 		return nil, err
+// 	}
+
+// 	// In certain edge cases, like when the L2 has fallen too far behind the L1, we are required to
+// 	// submit empty batches until we catch up.
+// 	if derive.NodeKitBatchMustBeEmpty(d.rollupCfg, l1Origin, batch.windowStart) {
+// 		batch.transactions = nil
+// 	}
+
+// 	attrs, err := d.attrBuilder.PreparePayloadAttributes(ctx, batch.onto, l1Origin.ID(), &batch.jst)
+// 	if err != nil {
+// 		return nil, err
+// 	}
+// 	attrs.NoTxPool = true
+// 	attrs.Transactions = append(attrs.Transactions, batch.transactions...)
+
+// 	// trigger next block production by javalin
+// 	// go func() {
+// 	// 	attrsEvent := &eth.BuilderPayloadAttributesEvent{
+// 	// 		Version: "",
+// 	// 		Data: eth.BuilderPayloadAttributesEventData{
+// 	// 			ProposalSlot:    l2Head.Number + 1,
+// 	// 			ParentBlockHash: l2Head.Hash,
+// 	// 			PayloadAttributes: eth.BuilderPayloadAttributes{
+// 	// 				Timestamp:             uint64(attrs.Timestamp),
+// 	// 				PrevRandao:            common.Hash(attrs.PrevRandao),
+// 	// 				SuggestedFeeRecipient: attrs.SuggestedFeeRecipient,
+// 	// 				GasLimit:              uint64(*attrs.GasLimit),
+// 	// 				// here we include zero transactions just to trigger javalin block production
+// 	// 				// javalin will fetch transactions from op-geth mempool
+// 	// 				Transactions: types.Transactions{},
+// 	// 			},
+// 	// 		},
+// 	// 	}
+
+// 	// 	attrsData, err := json.Marshal(attrsEvent)
+// 	// 	if err != nil {
+// 	// 		d.log.Error("failed to marshal payload attributes", "err", err)
+// 	// 	}
+// 	// 	d.broadcastPayloadAttrs("payload_attributes", attrsData)
+// 	// }()
+
+// 	d.log.Debug("prepared attributes for new NodeKit block",
+// 		"num", batch.onto.Number+1, "time", uint64(attrs.Timestamp), "origin", l1Origin)
+
+// 	// Start a payload building process.
+// 	withParent := derive.NewAttributesWithParent(attrs, batch.onto, false)
+// 	errTyp, err := d.engine.StartPayload(ctx, batch.onto, withParent, false)
+// 	if err != nil {
+// 		return nil, fmt.Errorf("failed to start building on top of L2 chain %s, error (%d): %w", batch.onto, errTyp, err)
+// 	}
+
+// 	// Immediately seal the block in the engine.
+// 	payload, errTyp, err := d.engine.ConfirmPayload(ctx, agossip, sequencerConductor)
+// 	if err != nil {
+// 		_ = d.engine.CancelPayload(ctx, true)
+// 		return nil, fmt.Errorf("failed to complete building block: error (%d): %w", errTyp, err)
+// 	}
+// 	d.nodekitBatch = nil
+// 	return payload, nil
+// }
+
+// func (d *Sequencer) cancelBuildingNodeKitBatch() {
+// 	// If we're in the process of building an NodeKit batch, we haven't sent anything to the engine
+// 	// yet. All we have to do is forget the batch.
+// 	d.nodekitBatch = nil
+// }
```

Comment:
would say here and up just delete the commented out code otherwise file looks good